backend/data/processors/manual_fixes/ENGGFixes.py

```diff
@@ -88,16 +86,6 @@ def ENGG_2997():
     return "72UOC && (3778 || 3061)"
 
 
-def ENGG_3060():
-    """
-    "original": "Pre-requisite: Completion of 66 UOC from an engineering degree<br/><br/>"
```

Comment:
Maybe we should at some point look at this years prerequisites vs last years to see if anything's changed. Only relevant for manual fixes I'd imagine

Reply:
I guess when whoever did this in 2021 the handbook said "Pre-requisite: Completion of 66 UOC from an engineering degree" and the 2022 now says "Pre-requisite: Completion of 66 UOC"

Reply:
CIRCLES-297 is a card for this, but it hasn't been picked up yet